### ISSUE TITLE
ChromeOS was not found on any device

### ISSUE THREAD

**Issue by BoBoBo51** [sebanc/brunch#592]
I'm going to install chrome os in my vmware, but this happened, someone help me please
![Capture](https://user-images.githubusercontent.com/73168353/96571195-738f7100-12f5-11eb-90f8-447ec2145bd4.PNG)

**Comment by sebanc** [comment]
Sorry but Brunch does not work in virtualbox due to missing graphic drivers as explained in #303

**Comment by BoBoBo51** [comment]
So it only works on real machines or what?

**Comment by sebanc** [comment]
yes only on the computers matching the specs detailed in the readme

**Comment by BoBoBo51** [comment]
Okay sebanc, now i installed it on a real machine but it still is the same, any answer?

**Comment by sebanc** [comment]
Chromeos not found message ?

**Comment by BoBoBo51** [comment]
Yes

**Comment by BoBoBo51** [comment]
Same message

**Comment by sebanc** [comment]
I will need a photo of the screen with chromeos not found message and your grub configuration

**Comment by BoBoBo51** [comment]
Okay 😔

**Comment by sebanc** [comment]
Also what's your laptop model ?

**Comment by BoBoBo51** [comment]
no im doing on a pc with a intel pentium dual core
![IMG_4693](https://user-images.githubusercontent.com/73168353/98465277-acd34680-21fa-11eb-8e94-0a95108fd070.JPG)

**Comment by sebanc** [comment]
As mentioned in the readme brunch is not compatible with intel cpu before 1st generation "Nehalem" (refer to https://en.wikipedia.org/wiki/Intel_Core)

**Comment by BoBoBo51** [comment]
So my laptop has 4th gen chip, will it work?

**Comment by sebanc** [comment]
yes 4th gen should work

**Comment by BoBoBo51** [comment]
ok let me try

**Comment by BoBoBo51** [comment]
Nah this time 4th gen cpu but idont have the internet when enter linux mint. Should i turn off secureboot or what

**Comment by BoBoBo51** [comment]
I5 4200U
4GB of ram
Sp uefi boot
I think this should work

**Comment by BoBoBo51** [comment]
hey sebanc

**Comment by sebanc** [comment]
It seems you are not that familiar with linux, you should probably follow the windows install procedure

**Comment by BoBoBo51** [comment]
What?

**Comment by sebanc** [comment]
Try to create the brunch usb flashdrive under windows, it might be easier:
https://github.com/sebanc/brunch#install-chromeos-on-a-usb-flash-drive--sd-card
